packages/components/src/templates/teaser-media.stories.ts

```diff
@@ -0,0 +1,106 @@
+import '../solid-components';
+import { html } from 'lit-html';
+
+export default {
+  tags: ['!dev'],
+  title: 'Templates/Teaser Media',
+  parameters: {
+    chromatic: { disableSnapshot: true }
+  }
+};
+
+/**
+ *
+ * - Aligment: Teaser contents can be center aligned if desired.
+ * - Paddings: Can be changed as desired.
+ * - Headline size: Can be changed as desired.
+ *
+ * ```
+ * ```
+ */
+
+export const Default = {
+  render: () =>
+    html` <style>
+        #part-content sd-teaser-media::part(content) {
+          padding: 60px;
+        }
+      </style>
+      <div class="flex flex-wrap gap-8 w-[600px]">
+        <div id="part-content">
+          <sd-teaser-media variant="gradient-dark" class="min-w-[435px] max-w-4xl">
+            <div slot="media" class="relative">
+              <img src="./placeholders/images/coffeeshop.jpg" alt="A skyline of a city" />
+            </div>
+            <h1 slot="headline" class="text-center text-4xl">Gender</h1>
+            <div slot="expandable" class="text-center">
+              <p>We are actively promoting gender equality.</p>
+            </div>
+          </sd-teaser-media>
+        </div>
+        <sd-teaser-media variant="gradient-dark" class="min-w-[435px] max-w-4xl">
+          <div slot="media" class="relative">
+            <img
+              class="aspect-video object-cover"
+              src="./placeholders/images/coffeeshop.jpg"
+              alt="A skyline of a city"
+            />
+          </div>
+          <div slot="meta" class="meta-info">
+            <span class="meta-info-item">August 2024</span>
```

Comment:
The text in this two span can be joined in a single span in my opinion

Reply:
Updated